runme.py

```diff
@@ -14,16 +14,15 @@
 BRANCH = "mozilla-release"
 REPO_PATH = "releases/{}".format(BRANCH)
 FROM = {
-    "version": "56.0",
-    "build_number": 6
-}
-TO = {
     "version": "56.0.1",
```

Comment:
Nit: I'm still waiting for the confirmation, but I think we want users to update from 56.0 to 56.0.2. People who are still on 32 bits after 56.0.1 MUST NOT get migrated (per cpeterson)

Reply:
sure thing, we should adjust these in any case